### PR TITLE
Add and Update channels

### LINKED ISSUE #8284: Add: AXN
### Channel ID

AXNChile.cl

### Stream URL

https://cvi.cdn.rcs.net.ar/mnp/axn_hls/playlist.m3u8

### Notes

I Don't Know If The Signal Is Chile Or Argentina 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8287: Add: TBN UK
### Channel ID

TBNUK.uk

### Stream URL

https://livestreamapis.com/v3/accounts/28567990/events/8919546/master.m3u8?token=0f6a2e07716d2b80af8d4ea3b03e9665&clientId=13929&timestamp=1661468530728

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8294: Add: The Danish Parliament
### Channel ID

TVfromtheDanishParliament.dk

### Stream URL

https://klive.kaltura.com/s/env/cluster-1-a.live.frp1/live/hls/p/2158211/e/1_24gfa7qq/tl/main/st/0/t/ys5JUGseQbAnuzkQ0G8Zzw/index-s32.m3u8?__hdnea__=st=1661400124~exp=1661486524~acl=/s/env/cluster-1-a.live.frp1/live/hls/p/2158211/e/1_24gfa7qq/tl/main/st/0/t/ys5JUGseQbAnuzkQ0G8Zzw/index-s*~hmac=47a364c53e08fac7f5ad8a2e4ed061610d6a322d6a809425b5f180b21b38a368

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8304: Add: Niederbayern TV Passau
### Channel ID

NiederbayernTVPassau.de

### Stream URL

https://stream03.welocal.stream/stream/fhd-nbypassau_49171/ngrp:stream_all/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8305: Add: Niederbayern TV Landshut
### Channel ID

NiederbayernTVLandshut.de

### Stream URL

https://stream03.welocal.stream:443/stream/fhd-nbylandshut_75262/ngrp:stream_all/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8306: Add: Niederbayern TV Deggendorf & Straubing
### Channel ID

NiederbayernTVDeggendorfStraubing.de

### Stream URL

https://stream03.welocal.stream:443/stream/fhd-nbydeggendorf_99148/ngrp:stream_all/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8307: Add: Niederbayern TV
### Channel ID

NiederbayernTV.de

### Stream URL

https://stream03.welocal.stream/stream/sat-nby/ngrp:sat-nby.stream_all/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8309: Add: ETB 1
### Channel ID

ETB1.es

### Stream URL

https://multimedia.eitb.eus/live-content/etb1hd-hls/master.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8310: Add: ETB 2
### Channel ID

ETB2.es

### Stream URL

https://multimedia.eitb.eus/live-content/etb2hd-hls/master.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8311: Add: ETB Basque
### Channel ID

ETBBasque.es

### Stream URL

https://multimedia.eitb.eus/live-content/eitbbasque-hls/master.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8312: Add: XON
### Channel ID

XON.nl

### Stream URL

https://ms7.mx-cd.net/tv/113-474263/EdeTV.smil/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8333: Add: KBEV
### Channel ID

KBEV.us

### Stream URL

https://s3-us-west-2.amazonaws.com/beverly-hills-high-school.castus-vod/live/ch1/video.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8334: Add: BHTV10
### Channel ID

BHTV10.us

### Stream URL

https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/beverlyhills/G0072_006/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8335: Add: BHTV35
### Channel ID

BHTV35.us

### Stream URL

https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/beverlyhills/G0072_007/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8337: Add: BPTV
### Channel ID

BPTV.us

### Stream URL

https://edge-f.swagit.com/live/buenaparkca/smil:std-4x3-1-a/chunklist_b512000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8342: Add: TVSN
### Channel ID

TVSN.au

### Stream URL

https://tvsnhlslivetest.akamaized.net/hls/live/2034711/TVSN-MSL4/master.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8346: Add: The Burbank Channel
### Channel ID

TheBurbankChannel.us

### Stream URL

https://cdn3.wowza.com/5/djRwZmQvTEJidmZD/burbank/G0240_009/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8347: Add: CTV
### Channel ID

CTV.us

### Stream URL

https://cdn3.wowza.com/5/UWpORHhLSEs5SkJs/calabasas/G0009_003/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8349: Add: Cerritos TV3
### Channel ID

CerritosTV3.us

### Stream URL

https://cdn3.wowza.com/5/UWpORHhLSEs5SkJs/cerritos/G0010_002/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8350: Add: City TV Channel 3/41
### Channel ID

CityTVChannel341.us

### Stream URL

https://edge-f.swagit.com/live/chinohillsca/smil:std-4x3-1-a/chunklist_b512000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by Alstruit**
Closes the following:

Fixes #8342, fixes #8312, fixes #8309, fixes #8310, fixes #8311, fixes #8307, fixes #8304, fixes #8305, fixes #8306, fixes #8287, fixes #8284.

Fixes #8294, fixes #8334, fixes #8333, fixes #8335, fixes #8337, fixes #8346, fixes #8347, fixes #8349, fixes #8350.

Update kr.m3u:
Fix naming typo
Add 2 channels
Fix 1 broken link

### CODE REVIEW

**Comment by UltraHDR** [comment]
Good job @Alstruit

**Review by UltraHDR — DISMISSED** [review]

**Comment by egypsiano** [comment]
Hi All,
Could you please Add these channels in Egypt (usually on Nilesat)
Cima, Al Hayaat Moslsalat, Oscar and Cairo Cinema
I really appreciate your efforts
thank you so much

File: streams/us.m3u [hunk 1]
```diff
@@ -1586,5 +1586,21 @@ https://d3qm7vzp07vxse.cloudfront.net/v1/master/77872db67918a151b697b5fbc23151e5
...
+#EXTINF:-1 tvg-id="" status="error",TV4 Bullhead City AZ (360p)
 https://cdn3.wowza.com/5/UWpORHhLSEs5SkJs/bullheadcity/G0860_001/playlist.m3u8
+#EXTINF:-1 tvg-id="KBEV.us" status="",KBEV 6 (1080p)
+https://s3-us-west-2.amazonaws.com/beverly-hills-high-school.castus-vod/live/ch1/video.m3u8
+#EXTINF:-1 tvg-id="BHTV10.us" status="",Beverly Hills Television 10 (720p)
+https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/beverlyhills/G0072_006/playlist.m3u8
+#EXTINF:-1 tvg-id="BHTV35.us" status="",Beverly Hills Television 35 (720p)
+https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/beverlyhills/G0072_007/playlist.m3u8
+#EXTINF:-1 tvg-id="BPTV.us" status="",Buena Park Television (360p)
+https://edge-f.swagit.com/live/buenaparkca/smil:std-4x3-1-a/chunklist_b512000.m3u8
```
Comment (LaneSh4d0w) [inline streams/us.m3u:1598]:
```suggestion
https://edge-f.swagit.com/live/buenaparkca/smil:std-4x3-1-a/playlist.m3u8
```

File: streams/us.m3u [hunk 2]
```diff
@@ -1586,5 +1586,21 @@ https://d3qm7vzp07vxse.cloudfront.net/v1/master/77872db67918a151b697b5fbc23151e5
 https://dai.google.com/linear/hls/event/RDmbn03GRCOLNG1eetEKhg/master.m3u8
 #EXTINF:-1 tvg-id="" status="error",AXN East Asia (Indonesian subtitle) (576p)
 http://210.210.155.37/dr9445/h/h141/index.m3u8
-#EXTINF:-1 tvg-id="" status="error",TV4 Bullhead City AZ
+#EXTINF:-1 tvg-id="" status="error",TV4 Bullhead City AZ (360p)
 https://cdn3.wowza.com/5/UWpORHhLSEs5SkJs/bullheadcity/G0860_001/playlist.m3u8
+#EXTINF:-1 tvg-id="KBEV.us" status="",KBEV 6 (1080p)
```
Comment (LaneSh4d0w) [inline streams/us.m3u:1591]:
```suggestion
#EXTINF:-1 tvg-id="KBEV.us" status="",KBEV 6 (1080p)
```

Website says Spectrum Channel 6. KBEV 6 is a student-run channel from Beverly Hills

**Review by BellezaEmporium — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**